Makefile-docker

```diff
@@ -77,6 +77,25 @@ update_assets:
 	$(PYTHON_COMMAND) manage.py collectstatic --noinput
 
 
+# Build list of dependencies to install
+DEPS = pip prod
+# If we're running a development image, then we should install the development dependencies
+ifeq ($(DOCKER_TARGET), development)
+DEPS += dev
+endif
+
+.PHONY: update_deps
+update_deps: ## Update the dependencies
+# If we're running a non local or production image, remove the existing deps and install clean
+ifneq ($(and $(findstring local, $(DOCKER_TAG)), $(findstring development, $(DOCKER_TARGET))),)
+	@echo "Updating existing deps"
+else
+	@echo "Removing existing deps"
+	rm -rf /deps/**
+endif
```

Comment:
similarly these lines don't appear to be indented

Reply:
conditionals inside of recipes should not be indented